suites/upgrade/infernalis/older/3-upgrade-sequence/upgrade-mon-osd-mds.yaml

```diff
@@ -0,0 +1,44 @@
+meta:
+   - desc: |
+      upgrade mon a and b
```

Comment:
could simplify restart order description actually - I'd suggest 'restart monitors, mds, and then osds'